### ISSUE TITLE
unexpected error causes a11y scan action to fail

### ISSUE THREAD

**Issue by scottaohara** [github/accessibility-scanner#22]
After a lot of trial and error after getting multiple instances of the a11y scanner failing against a long list of URLs, I discovered that if [this page (though copied into a different repo)](https://scottaohara.github.io/testing/failing-cookie-region.html) is included in the list of URLs to scan, it causes the action to fail with the following message:

```
/home/runner/work/_temp/583e4e7e-ec11-43e3-9b15-3d2b6b0f2c17.sh: line 2: syntax error near unexpected token `('
Error: Process completed with exit code 2.
```

When this happens, the action will create github issues for all the detections that were found in the scan (for all files prior to the action failing), but since the action errors out, copilot never kicks off to start working on the issues that are opened.  If someone tries re-running the action, duplicate issues will be logged even though they were already logged in the previous failed scan(s).

Hopefully this is enough information to look into this so a fix can be made.

**Comment by smockle** [comment]
Thanks for reporting this @scottaohara! I appreciate you including a page (https://scottaohara.github.io/testing/failing-cookie-region.html) where this repros; that made this much easier to debug. I opened https://github.com/github/accessibility-scanner/pull/36 with a fix. I’ll keep this issue open until that fix is backported to the v1 and v2 series.
